src/app/services/work-item.service.ts

```diff
@@ -779,16 +773,14 @@ export class WorkItemService {
       opLinkTypes.push({
         name: linkType.attributes['forward_name'],
         linkId: linkType.id,
-        linkType: 'forward',
-        wiType: linkType.relationships.target_type.data.id
+        linkType: 'forward'
       });
     });
     linkTypes.backwardLinks.forEach((linkType: LinkType) => {
       opLinkTypes.push({
         name: linkType.attributes['reverse_name'],
         linkId: linkType.id,
-        linkType: 'reverse',
-        wiType: linkType.relationships.source_type.data.id
```

Comment:
Because it no longer exists, since the `source_type` field has been removed from the link type.